src/cmd.rs

```diff
@@ -1,26 +1,24 @@
 use std::process::Command;
```

Comment:
We don't need the `call` variants without paths any more since cargo is not required.